task-2/.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+	"trailingComma": "all",
+	"printWidth": 80,
+	"tabWidth": 2,
+	"semi": false,
```

Comment:
Let's make it "true" since it's a best practice anyway.

Reply:
.prettierrc file has been updated.